### DISCUSSION TITLE
Silage making without Autodrive (HAWE suw5000)

### DISCUSSION THREAD

**Discussion by rob230580** [Courseplay/Courseplay_FS22#2692] [Q&A]
Hello,

I'm trying to make silage without needing Auto-drive, so I got the fermenting silo mod to take bunker filling out of the equation, using AI workers to take chaff from the field to the silo. IE no reversing trailers, compacting etc.

I tried to use the HAWE suw5000 as an over-loader to take chaff from the forager and unload to the AI trailer on the field edge, which is what I do with harvesting. Unfortunately course-play doesn't recognise the HAWE trailer as an over-loader and maybe not the forager as a combine for this mode?

Am I doing something wrong? Or is it just not possible to work this way?

Thanks

**Comment by Tensuko** [comment]
The HAWE would also not work as one.
That tool needs special treatment to work.
Not sure if we will get this to work.

**Comment by rob230580** [comment]
I thought so, i guess the mechanics are different. Thanks anyway
